var/spack/repos/builtin/packages/rose/package.py

```diff
@@ -85,17 +93,46 @@ def languages(self):
 
     def configure_args(self):
         spec = self.spec
-        cc = self.compiler.cc
-        cxx = self.compiler.cxx
+
+        if '+intel_backend' in spec:
+            cc = which('mpicc')
+            cxx = which('mpic++')
```

Comment:
I don't think this is the right thing to do. If you add an MPI dependency, you can use `cc = spec['mpi'].mpicc`.